src/components/ArticleController.jsx

```diff
@@ -15,7 +15,7 @@ export default class ArticleController extends FalcorController {
       let articleCategory = this.props.params.articleCategory;
       let articleSlug = this.props.params.articleSlug;
       // Access data fetched via Falcor
-      const articleData = this.state.data.issues[issueId]["articles"][articleCategory][articleSlug];
+      const articleData = this.state.data.issues[issueId].articles[articleCategory][articleSlug];
```

Comment:
Zane and I already talked about it in our Google Hangout and he already changed it, because you are exactly right :)